.drone.yml

```diff
@@ -49,6 +49,7 @@ build:
       - go get github.com/golang/lint/golint
       - go get -u gopkg.in/check.v1
       - make -s build
+      - make -s build-vfile
```

Comment:
it is better to include the target into the `build` target it self

Reply:
If we include vFile also into build target, all the following development for vDVS will suffer waiting for building vfile too. The build of vfile and vdvs are totally separate from each other. So I think we should have them as different build target.

Reply:
building vFile binary wont take much time it is still OK to build binary as part of build having it said building and pushing plugin should be a separate target and I think it is there. Am I missing anything here?